cardano_node_tests/tests/test_pools.py

```diff
@@ -87,6 +88,109 @@ def cluster_mincost(
     )
 
 
+def _check_pool_deregistration_in_db(
```

Comment:
I added check functions: `_check_pool_deregistration_in_db` and `_check_pool_data_in_db`  to this test file as I wasn't sure if we want to use them in any place outside of it - I think no, (at least for now). if needed they can be moved to `clusterlib utils`